test/ReadsAlignmentSet_basic_test.py

```diff
@@ -7,12 +7,14 @@
 from os import environ
 from pprint import pprint
 
+from test import TEST_BASE_DIR
 from SetAPI.SetAPIImpl import SetAPI
 from SetAPI.SetAPIServer import MethodContext
-from SetAPI.authclient import KBaseAuth as _KBaseAuth
+from installed_clients.authclient import KBaseAuth
 from installed_clients.FakeObjectsForTestsClient import FakeObjectsForTests
 from installed_clients.WorkspaceClient import Workspace as workspaceService
-from util import (
+from test import TEST_BASE_DIR
```

Comment:
Duplicates line 10 above.

Reply:
This duplication has been removed in the next PR if you don't mind waiting until then.